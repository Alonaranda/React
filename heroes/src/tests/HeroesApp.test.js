import { HeroesApp } from "../HeroesApp";
import React from 'react';
import {shallow} from 'enzyme';

describe('Probando HeroesApp', () => {
    test('Se muestra bien', () => {
        const wrapper = shallow(<HeroesApp/>);
        expect(wrapper).toMatchSnapshot();

    });
});