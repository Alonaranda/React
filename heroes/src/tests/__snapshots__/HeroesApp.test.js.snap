// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando HeroesApp Se muestra bien 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "user": Object {
        "logged": false,
      },
    }
  }
>
  <AppRouter />
</ContextProvider>
`;
